detection-rules/attachment_any_html_unsolicited.yml

```diff
@@ -12,10 +12,10 @@ source: |
   type.inbound
   and any(attachments, .file_extension in~ ('htm', 'html') or .file_type == "html")
   and (
-    not profile.by_sender().solicited
-    or (
+    not profile.by_sender().any_false_positives
```

Comment:
We need to keep unsolicited as a requirement

Reply:
I think this is fixed? Feel free to just overwrite my changes.

Reply:
It looks fixed to me, what am I missing?

Reply:
Nothing, it's just that my brain isn't active yet... so the odds of an error are higher that usual 😛

Reply:
Oh sorry, I was just making sure @rw-access doesn't have an additional concern with the above. The rule looks like it's keeping unsolicited as a requirement, so I was confused by the comment.

Reply:
looks good now!